src/index/ivf/ivf.cc

```diff
@@ -541,17 +541,26 @@ IvfIndexNode<DataType, IndexType>::Search(const DataSet& dataset, const Config&
                             distances[i + offset] = static_cast<float>(i_distances[i + offset]);
                         }
                     }
-                } else if constexpr (std::is_same<IndexType, faiss::IndexIVFFlat>::value) {
+                } else if constexpr (std::is_same<IndexType, faiss::IndexIVFFlatCC>::value) {
```

Comment:
We're not using `IndexIVFFlat`, just `IndexIVFFlatCC` all the time, right?

Reply:
Actually I dont know why we need a seperate code branch for IndexIVFFlat, it is the same with IVFPQ, IVFSQ, etc

Reply:
But we do need a new code branch for IndexIVFFlatCC, so I replace it.